pkgs/by-name/co/comet-gog/package.nix

```diff
@@ -5,33 +5,31 @@
   protobuf,
 }:
 
-rustPlatform.buildRustPackage rec {
+rustPlatform.buildRustPackage (finalAttrs: {
   pname = "comet-gog";
   version = "0.3.1";
 
   src = fetchFromGitHub {
     owner = "imLinguin";
     repo = "comet";
-    tag = "v${version}";
+    tag = "v${finalAttrs.version}";
     hash = "sha256-asg2xp9A5abmsF+CgOa+ScK2sQwSNFQXD5Qnm76Iyhg=";
     fetchSubmodules = true;
   };
 
   cargoHash = "sha256-K0lQuk2PBwnVlkRpYNo4Z7to/Lx2fY6RIlkgmMjvEtc=";
 
-  # error: linker `aarch64-linux-gnu-gcc` not found
-  postPatch = ''
-    rm .cargo/config.toml
-  '';
```

Comment:
Have you tried building on `aarch64-linux`?
This was needed for it.

(Not sure why nixpkgs review isn't building this package on non-`x86_64-linux` systems)

Reply:
No, I'm not set up to build nixpkgs for other architectures.  It wasn't clear from the comment whether this error was happening on all architectures.  Can we make it an `optionalString` on whether the platform is `aarch64-linux`?

Reply:
All packages do this unconditionally currently in nixpkgs.

One package has this comment:
```
    # Upstream started to redirect aarch64-unknown-linux-gnu to aarch64-linux-gnu-gcc
    # for their CI which breaks compiling on aarch64 in nixpkgs:
    #  error: linker `aarch64-linux-gnu-gcc` not found
```

If you want to, you can adapt this somehow.

Reply:
Reverted.